curvefs/src/mds/fs_manager.cpp

```diff
@@ -57,20 +57,28 @@ FSStatusCode FsManager::CreateFs(const std::string& fsName,
     // 1. query fs
     // TODO(cw123): if fs status is FsStatus::New, here need more consideration
     if (fsStorage_->Exist(fsName)) {
-        if (!IsPreviousCreateUnComplete(fsName, fsType, blockSize, detail)) {
+        int existRet = IsPreviousCreateUnCompleteOrExist(fsName, fsType,
```

Comment:
it's better to rename it to `IsExactlySameOrCreateUnComplete`

Reply:
> it's better to rename it to `IsExactlySameOrCreateUnComplete`

fixed